templates/conf.py_t

```diff
@@ -65,7 +65,7 @@ master_doc = '{{ master_str }}'
 
 # General information about the project.
 project = u'{{ project_str }}'
-copyright = u'{{ copyright_str }}'
+copyright = 'FOSSASIA'
```

Comment:
Is this really needed? copyright_str seems to automatically generated by sphinx in the format YEAR, AUTHOR